.gitlab-ci.yml

```diff
@@ -549,3 +542,49 @@ publish:s3:mender-gateway:automatic:
   rules:
     - if: '$PUBLISH_MENDER_DIST_PACKAGES_AUTOMATIC == "true"'
   extends: .template:publish:s3:mender-gateway
+
+.template:publish:s3:latest:manual:
+  when: manual
+  stage: publish
+  image: debian:buster
+  dependencies:
+    - build:packages:debian:buster:amd64
+    - build:packages:debian:buster:armhf
+    - build:packages:debian:buster:arm64
+    - build:packages:debian:bullseye:amd64
+    - build:packages:debian:bullseye:armhf
+    - build:packages:debian:bullseye:arm64
+    - build:packages:ubuntu:bionic:amd64
+    - build:packages:ubuntu:bionic:armhf
+    - build:packages:ubuntu:bionic:arm64
+    - build:packages:ubuntu:focal:amd64
+    - build:packages:ubuntu:focal:armhf
+    - build:packages:ubuntu:focal:arm64
+  before_script:
+    - apt update && apt install -yyq awscli
+    - *publish_helper_functions
+  script:
+    - echo "Publishing "latest" package for ${PUBLISH_PACKAGE_PREFIX} to s3://${S3_BUCKET_NAME_PRIVATE}/${PUBLISH_PACKAGE_S3_SUBPATH}/${PUBLISH_PACKAGE_VERSION}/"
+    - if ! is_final_tag ${PUBLISH_PACKAGE_VERSION}; then
```

Comment:
paranoia mode on: I would double quote the version here.

Reply:
Sure, I'll do that.